tests/unit/test_bot.py

```diff
@@ -70,23 +79,39 @@ async def test_extracts_urls(self, bot: OdesliBot):
         )
         spotify = urls[9]
         assert spotify.platform_key == 'spotify'
-        assert spotify.url == 'https://link.tospotify.com/pfc3erwl2ab'
+        assert spotify.url == (
+            'https://link.tospotify.com/pfc3erwl2ab'
+        )
```

Comment:
Are these conversions to multi-line required?

Reply:
Not required, but the combination of multi-line and single-line is difficult to read.

Reply:
OK, but again:
1. It's not related to the PR subject.
2. Let's the maintainer to decide is it in their taste (because "readability" is pretty subjective thing), but judging on how it was initially… doubtful.